test/e2e/initialization/config.go

```diff
@@ -384,3 +390,28 @@ func updateGenUtilGenesis(c *internalChain) func(*genutiltypes.GenesisState) {
 		genUtilGenState.GenTxs = genTxs
 	}
 }
+
+func applyRateLimitsToChainConfig(rateLimiterGenState *ratelimiter.GenesisState) {
+	path := &ratelimiter.Path{
+		Denom:     "ubbn",
+		ChannelId: "channel-0",
+	}
+
+	quota := &ratelimiter.Quota{
+		MaxPercentSend: sdkmath.NewInt(90),
+		MaxPercentRecv: sdkmath.NewInt(90),
+		DurationHours:  24,
+	}
+
+	rateLimit := ratelimiter.RateLimit{
+		Path:  path,
+		Quota: quota,
+		Flow: &ratelimiter.Flow{
+			Inflow:       sdkmath.NewInt(0),
+			Outflow:      sdkmath.NewInt(0),
+			ChannelValue: sdkmath.NewInt(1_000_000),
```

Comment:
we could increase this channel value to at least 1M ≃ `1_000_000000` as it represents the total supply of the token in the host chain

Reply:
yeah, the thing is that will need to fund the sender account with enough funds to reach the limit for the quota exceeded case